infra/resources/modules/commons/apim.tf

```diff
@@ -246,3 +246,33 @@ resource "azurerm_api_management_subscription" "io_backend" {
   state               = "active"
   allow_tracing       = false
 }
+
+####################################################################################
+# Support User
+####################################################################################
+resource "azurerm_api_management_user" "support_user" {
+  user_id             = "support-user"
+  api_management_name = module.apim.name
+  resource_group_name = module.apim.resource_group_name
+  first_name          = "Support"
+  last_name           = "PagoPA"
+  email               = "trialsystem-support@pagopa.it"
```

Comment:
**question**: @AleDore do we need to add an email address of our support team or can we use an alias of our email address? (e.g. trialsystem+support@pagopa.it)

Reply:
we can change it later. We can keep this one at the moment :)